### ISSUE TITLE
Problems deploying Alfresco CE in K8s cluster

### ISSUE THREAD

**Issue by dgarcoe** [Alfresco/acs-deployment#569]
Hi,

I'm trying to deploy the community edition in a K8s cluster from a partner but so far I'm not able to do it. I've been following the instructions in the Docker for Desktop page. This is what I'm doing:

~~~
$kubectl create namespace alfresco

$helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
$helm repo update

$helm install acs-ingress ingress-nginx/ingress-nginx --version=3.7.1 --set controller.scope.enabled=true --set controller.scope.namespace=alfresco --set rbac.create=true --atomic --namespace alfresco

$helm repo add alfresco https://kubernetes-charts.alfresco.com/stable
$helm repo update

$helm install acs alfresco/alfresco-content-services --values=community_values.yaml --set externalPort="80" --set externalProtocol="http" --set externalHost="localhost" --atomic --timeout 10m0s --namespace=alfresco

~~~

However, I'm getting this error that seems to be related to the ingress controller. In fact, I have seen another similar issue that was solved by pinning the ingress version. Although it's not the case right now.

~~~
Error: exit status 1
2021/06/30 14:55:56 [emerg] 1864#1864: location "^(/.*/proxy/.*/api/solr/.*)$" cannot be inside the exact location "/share" in /tmp/ng
inx-cfg284669244:521
nginx: [emerg] location "^(/.*/proxy/.*/api/solr/.*)$" cannot be inside the exact location "/share" in /tmp/nginx-cfg284669244:521
nginx: configuration file /tmp/nginx-cfg284669244 test failed
~~~

Thanks for your help!

EDIT: I updated the error message as the first one was not the correct one.

**Comment by bstrdsmkr** [comment]
@dgarcoe see: https://github.com/Alfresco/acs-deployment/issues/460

tl;dr -- Alfresco requires an old version of ingress-nginx :(

**Comment by mteodori** [comment]
@bstrdsmkr doesn't seem to be the case as @dgarcoe is installing the suggested version 3.7.1

**Comment by mteodori** [comment]
that error shows with the latest ingress not the 3.7.1 we suggested, now it's fixed on master but there are still some other issues with the community build we will fix before releasing 5.2.0

**Comment by mteodori** [comment]
5.2.0 is now released supporting the latest https://github.com/kubernetes/ingress-nginx
